docs/bcfusion/validators/gov.md

```diff
@@ -0,0 +1,102 @@
+# Governance
+
+[BEP-297](https://github.com/bnb-chain/BEPs/pull/297) introduces the native governance module for BNB smart chain after
+BNB chain fusion. The governance module is derived
+from [OpenZeppelin Governor](https://docs.openzeppelin.com/contracts/4.x/governance), with the following
+features:
+
+- Any staking credit holder is allowed to propose and vote.
+- Voters can continue to earn staking rewards during the voting period.
+- Users can delegate someone else to participate in governance on their behalf.
+- A time lock period is introduced before the proposal is executed.
+- If the proposal is rejected, the proposer does not incur any financial losses.
+
+## Workflow
+
+The workflow of the governance module consists of three stages: submit proposal, cast vote, and execute proposal. Each
+stage has its own requirements and parameters, which can be configured by the BNB smart chain governance parameters.
+
+### Submit Proposal
+
+To submit a proposal, a staking credit holder needs to send a `propose` transaction to the BNB smart chain, specifying
+the following information:
+
+- **Proposer address**: The BEP20 address of the proposer, who initiates the proposal and pays the proposal fee.
+- **Targets**: The list of addresses of the contracts or accounts that the proposal wants to interact with.
+- **Values**: The list of amounts of BNB or other tokens that the proposal wants to transfer to the targets.
+- **Signatures**: The list of function signatures of the contracts that the proposal wants to call on the targets.
+- **Calldatas**: The list of encoded arguments of the functions that the proposal wants to call on the targets.
+- **Description**: The description of the proposal, which provides more details and rationale for the proposal.
+
+A delegator should delegate more the 200 staked BNB as the minimal requirement for submitting a proposal.
+The `propose` transaction will deduct the proposal fee (TODO: currently there is no such fee, we may add the fee) 
```

Comment:
let us remove the fee part here first.

Reply:
sure, removed

Reply:
We change the logic to: one user can only have one active/pending proposal now.

Reply:
updated.